### PR TITLE
{lib}[GCC/11.2.0] c-ares v1.18.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    '414872549eec4e221b576693fdc9c9bce44ff794d0f1f06f2515b56a7f6ec9c9',  # cares-1_18_1.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('pkg-config', '0.29.2'),
+    ('CMake', '3.22.1'),
+]
+
+parallel = 2
```
Comment (boegel) [inline easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb:31]: @sassy-crick Any reason to hardcode this?

File: easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
```
Comment (boegel) [inline easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb:3]: @sassy-crick Any reason to go with `CMakeMake`? Existing easyconfigs for `c-ares` v1.18.1 are using `ConfigureMake`, should we consider changing those as well (if so, let's make that a separate PR)

File: easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/c-ares/c-ares/archive']
+sources = ['cares-%s.tar.gz' % version.replace('.', '_')]
+
+checksums = [
+    '414872549eec4e221b576693fdc9c9bce44ff794d0f1f06f2515b56a7f6ec9c9',  # cares-1_18_1.tar.gz
+]
```
Comment (boegel) [inline easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb:23]: This can/should be trimmed since there's only a single source file:

```python
source_urls = ['https://github.com/c-ares/c-ares/archive']
sources = ['cares-%s.tar.gz' % version.replace('.', '_')]
checksums = ['414872549eec4e221b576693fdc9c9bce44ff794d0f1f06f2515b56a7f6ec9c9']

File: easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    ('binutils', '2.37'),
+    ('pkg-config', '0.29.2'),
+    ('CMake', '3.22.1'),
+]
+
+parallel = 2
+
+sanity_check_paths = {
+    'files': ['lib/libcares.%s' % SHLIB_EXT],
+    'dirs': ['include', 'share'],
```
Comment (boegel) [inline easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb:35]: Existing easyconfigs have a slightly more extensive set of paths to check, please sync up?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb:31]: The idea was to not try and build it with too many cores but I have removed that.

File: easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb (hunk 2 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb:3]: I think that was my wrong. I somehow did not find `c-ares` when I was looking for it so I created a new EC, hence the different description and the Author which I only do when it is a new PR, or a significant change to an existing PR. So when I saw `CMakeList.txt` I decided to go for 'the more modern' `CMake` instead of the more traditional `configure`. 
Given there is already the correct `c-ares` EC in place, we can close this PR.

**Comment by sassy-crick** [comment]
Surplus as there is already an existing `c-ares` EC file in place. Apologise for opening a duplicate. 
Closed.
